plugins/brjs-plugins/src/test/java/org/bladerunnerjs/spec/aliasing/AppLevelAliasOverrideTest.java

```diff
@@ -0,0 +1,382 @@
+	private AliasDefinitionsFileBuilder aspectResourcesAliaseDefinitionsFileBuilder;
+	private AliasDefinitionsFileBuilder brLibAliasDefinitionsFileBuilder;
+	private SdkJsLib servicesLib;
+	private StringBuffer response = new StringBuffer();
+	private StringBuffer responseWithoutAliasesFile = new StringBuffer();
+	private AliasDefinitionsFileBuilder bladeAliasDefinitionsFileBuilder;
+	private Bladeset bladeset;
+	private Blade blade, bladeForWorkbenchWithoutAliasesFile;
+	private BladeWorkbench workbench;
+	private AliasesFile worbenchAliasesFile;
+	private Aspect aspectWithoutAliasesFile;
+	private BladeWorkbench workbenchWithoutAliasesFile;
+	private TestPack bladeTestPack;
+	private AliasesFileBuilder bladeTestPackAliasesFileBuilder;
+	private AliasDefinitionsFileBuilder bladesetResourcesAliasDefinitionsFileBuilder;
+	private AliasDefinitionsFileBuilder bladeResourcesAliasDefinitionsFileBuilder;
+	
+	@Before
+	public void initTestObjects() throws Exception
+	{
+		given(brjs)
+			.automaticallyFindsBundlerPlugins()
+			.and(brjs).automaticallyFindsMinifierPlugins()			
+			.and(brjs).hasBeenCreated();
+		app = brjs.app("app1");
+		appConf = app.appConf();
+		aspect = app.aspect("default");
+		aspectWithoutAliasesFile = app.aspect("withoutAliasesFile");
+		bladeset = app.bladeset("bs");
+		blade = bladeset.blade("b1");
+		bladeTestPack = blade.testType("ut").defaultTestTech();
+		bladeForWorkbenchWithoutAliasesFile = bladeset.blade("b2");
+		brLib = app.jsLib("br");
+		workbench = blade.workbench();
+		workbenchWithoutAliasesFile = bladeForWorkbenchWithoutAliasesFile.workbench();
+		worbenchAliasesFile = aliasesFile(workbench);
+		workbenchAliasesFileBuilder = new AliasesFileBuilder(this, worbenchAliasesFile);
+		appAliasesFileBuilder = new AliasesFileBuilder(this, aliasesFile(app));
+		aspectAliasesFileBuilder = new AliasesFileBuilder(this, aliasesFile(aspect));
+		brLibAliasDefinitionsFileBuilder = new AliasDefinitionsFileBuilder(this, aliasDefinitionsFile(brLib, "resources"));
+		aspectResourcesAliaseDefinitionsFileBuilder = new AliasDefinitionsFileBuilder(this, aliasDefinitionsFile(aspect, "resources"));
+		bladesetResourcesAliasDefinitionsFileBuilder = new AliasDefinitionsFileBuilder(this, aliasDefinitionsFile(bladeset, "resources"));
+		bladeResourcesAliasDefinitionsFileBuilder = new AliasDefinitionsFileBuilder(this, aliasDefinitionsFile(blade, "resources"));
+		bladeAliasDefinitionsFileBuilder = new AliasDefinitionsFileBuilder(this, aliasDefinitionsFile(blade, "src"));
+		bladeTestPackAliasesFileBuilder = new AliasesFileBuilder(this, aliasesFile(bladeTestPack));
+		
+		servicesLib = brjs.sdkLib("ServicesLib");
+		given(servicesLib).containsFileWithContents("br-lib.conf", "requirePrefix: br")
+			.and(servicesLib).hasClasses("br/AliasRegistry", "br/ServiceRegistry");
+	}
+	
+	@Test
+	public void aliasDefinitionsInLibsAreOverridenByTheAppAliases() throws Exception {
```

Comment:
These tests are psuedo BDD since `brLibAliasDefinitionsFileBuilder` and `appAliasesFileBuilder` contain technical information about the underlying type &mdash; that it's a builder. We should update all tests to remove anything like this.